app/styles/typography.css

```diff
@@ -0,0 +1,95 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@font-face {
+    font-family: 'Acronym';
+    src: url('../../public/fonts/Acronym/ACRONYM\ THIN.woff2') format('woff2');
+    font-weight: 100;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Acronym';
+    src: url('../../public/fonts/Acronym/ACRONYM\ ULTRALIGHT.woff2') format('woff2');
+    font-weight: 200;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Acronym';
+    src: url('../../public/fonts/Acronym/ACRONYM\ LIGHT.woff2') format('woff2');
+    font-weight: 300;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Acronym';
+    src: url('../../public/fonts/Acronym/ACRONYM\ REGULAR.woff2') format('woff2');
+    font-weight: 400;
+    font-style: normal;
+}
+
+/*no medium*/
+
+@font-face {
+    font-family: 'Acronym';
+    src: url('../../public/fonts/Acronym/ACRONYM\ SEMIBOLD.woff2') format('woff2');
+    font-weight: 600;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Acronym';
+    src: url('../../public/fonts/Acronym/ACRONYM\ BOLD.woff2') format('woff2');
+    font-weight: 700;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Acronym';
+    src: url('../../public/fonts/Acronym/ACRONYM\ EXTRABOLD.woff2') format('woff2');
+    font-weight: 800;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Acronym';
+    src: url('../../public/fonts/Acronym/ACRONYM\ BLACK.woff2') format('woff2');
+    font-weight: 900;
+    font-style: normal;
+}
+
+@layer base {
+    * {
+        @apply font-Acronym text-base;
```

Comment:
Acronym is not defined in the `tailwind.config.js`. you should use `font-acronym`. The line `"acronym": ["Acronym", "sans-serif"]" means `custom-font-name: [font-family, fallback]`.

![image](https://github.com/Himati-UP-Mindanao/himati-website/assets/85869308/56245415-2420-46a3-aede-44edfb54471d)